src/Agent.Worker/Container/ContainerInfo.cs

```diff
@@ -135,6 +182,78 @@ public string TranslateToHostPath(string path)
 
             return path;
         }
+
+        private List<MountVolume> ParseVolumes(IList<string> volumes)
+        {
+            List<MountVolume> mountVolumes = new List<MountVolume>();
+            if (volumes?.Count > 0)
+            {
+                foreach (var volume in volumes)
+                {
+                    var volumeSplit = volume.Split(":");
+                    if (volumeSplit?.Length == 3)
```

Comment:
not sure `volumeSplit` will ever be null.

Reply:
Updated